3-reveal/migrations/5-IRS/3-Namibia-2019/deploy/finish_namibia_irs_structures_redo.psql

```diff
@@ -0,0 +1,24 @@
+-- Deploy reveal_irs_namibia_2019:finish_namibia_irs_structures_redo to pg
+-- requires: redo_namibia_irs_structures
+-- requires: redo_dependencies_of_namibia_irs_structures
+
+BEGIN;
+
+SET search_path TO :"schema",public;
```

Comment:
any reason for dividing this into two sections?  I guess it's a little safer in that we're sure we won't impact superset?

Reply:
its mainly because I didn't want to have to change anything in Superset for Namibia